content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -1,75 +1,80 @@
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+To add multiple addresses to a contact, go to :menuselection:`Sales app --> Orders --> Customers`,
+and clear any default filters from the search bar. Then, click on the desired customer to open their
+contact form.
 
-.. image:: different_addresses/addresses_2.png
+.. tip::
+   Contact forms can be accessed in the *Contacts* application, as well.
+
+To add multiple addresses to a contact, navigate to the :menuselection:`Contacts` app, or go to
+:menuselection:`Sales app --> Orders --> Customers`, and clear any default filters from the search
+bar, then click on the desired customer to open their contact form.
+
+From the contact form, click :guilabel:`Edit`, then select :guilabel:`Add`, located under the
+:guilabel:`Contacts & Addresses` tab. Doing so reveals a :guilabel:`Create Contact` pop-up form, in
+which additional addresses can be configured.
+
+.. image:: different_addresses/contact-form-add-address.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to add addresses from a quotation on Odoo Sales?
+   :alt: Add a contact/address to the contact form.
 
-.. note::
-   If you select a customer with defined invoice and delivery addresses, Odoo will automatically use
-   them to fill in the fields. Now, if you want to change it instantly, it is possible to do so
-   directly from the quotation or the sales order.
+On the :guilabel:`Create Contact` pop-up form, start by clicking the default :guilabel:`Other
+Address` field to reveal a drop-down menu of address-related options.
 
-Add addresses from a contact form
-=================================
+Select any of the following options:
 
-Previously, we talked about the contact form that you can fill in directly from a quotation or a
-sales order to add billing and shipping addresses to customers. But, if you go to
-:menuselection:`Sales --> Orders --> Customers`, you can create or modify every customer you want
-and add, delete or modify invoice and delivery addresses instantly there, before creating a
-quotation.
+- :guilabel:`Contact`
+- :guilabel:`Invoice Address`
+- :guilabel:`Delivery Address`
+- :guilabel:`Other Address`
+- :guilabel:`Private Address`
 
-.. image:: different_addresses/addresses_3.png
+Once an option is selected, proceed to enter the corresponding contact information that should be
+used for the specified address type.
+
+.. image:: different_addresses/create-contact-window.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to add addresses from a contact form on Odoo Sales?
+   :alt: Create a new contact/address on a contact form.
+
+Then, click :guilabel:`Save & Close` to save the address and close the :guilabel:`Create Contact`
```

Comment:
Checking: is `Create Contact` the name of the window if we're editing an _existing_ contact?

We start by first instructing to _edit an existing contact_ (which seems more likely), and then it goes into a _new contact_ flow. Just making sure the flows are the same with same headings...

Reply:
Great question. Yes, :guilabel:`Create Contact` is the title of the pop-up window that appears while editing an *existing* contact (clicking 'Add' under the 'Contacts and Addresses' tab on the Contact form)